src/core/file.js

```diff
@@ -36,6 +37,16 @@ module.exports = {
     });
   },
 
+  move(filePath, targetDir, extraFile, logger = defaultLogger) {
```

Comment:
this function should only have 2 parameters: srcFilePath and targetFilePath
I don't think it should have extraFile

Reply:
Yeah, I removed the extraFile param in move function